### PR TITLE
MAT-8440: LibraryHistory visibility

### PR DESCRIPTION
**Pull request by ethankaplan**
## MADiE PR

Jira Ticket: [MAT-8440](https://jira.cms.gov/browse/MAT-8440)
(Optional) Related Tickets:

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included
* [ ] No extemporaneous files are included (i.e Complied files or testing results)
* [ ] This PR is into the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: src/components/PageHeader/PageHeader.tsx [hunk 1]
```diff
@@ -244,7 +244,7 @@ const PageHeader = () => {
       {pathname.includes("edit") && pathname.includes("cql-libraries") && (
         <Fade in={libraryState?.cqlLibraryName !== undefined}>
           <div className="details">
-            {libraryCanEdit && (
+            {(libraryCanEdit || featureFlags?.LibraryHistory) && (
```
Comment (sb-prateekkeerthi) [inline src/components/PageHeader/PageHeader.tsx:249]: I don't see any featureFlags variable declared in the file how are you being able to access it. I am seeing issues in the local

**Review by sb-prateekkeerthi — CHANGES_REQUESTED** [review]

File: src/components/PageHeader/PageHeader.tsx (hunk 1 above)
Comment (sb-prateekkeerthi) [reply src/components/PageHeader/PageHeader.tsx:249]: I think you missed committing the code for adding const featureFlags = useFeatureFlags(); in this file

**Review by sb-prateekkeerthi — APPROVED** [review]
